src/otx/core/model/entity/utils/mmseg.py

```diff
@@ -1,23 +1,124 @@
 # Copyright (C) 2024 Intel Corporation
+# Copyright (c) OpenMMLab. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 #
+# Please note that parts of source code here are
+# borrowed from https://github.com/open-mmlab/mmsegmentation
+#
 """Utility files for the mmseg package."""
-
+# ruff: noqa
```

Comment:
Can we remove this line?

Reply:
No, too many violations in the borrowed codes